.github/renovate.json5

```
@@ -94,6 +94,14 @@
 			matchPackageNames: [ '@wordpress/dataviews' ],
 			separateMajorMinor: false,
 			prPriority: 1,
+			// Teams using DataViews in their projects
+			additionalReviewers: [
+				'team:jetpack-reach', // Publicize
+				'team:loop', // Newsletter subscriber management
```

Loop seems wrong per https://github.com/Automattic/jetpack/pull/42806#issuecomment-2770604132

Probably should be
```suggestion
				'team:io', // Newsletter subscriber management
```
although I note GitHub doesn't seem to want to let me add that team as a reviewer manually so we probably need to configure something somewhere.

---

Loop team normally owns the Newsletter features and is most knowledgeable about subscriber management regressions, while Jorge from I/O was working on migration to DataViews specifically — so long-term likely best to keep Loop here, and they can add more folks if needed case-by-case? 

@allilevine could you confirm / suggest anything else?

---

That's accurate, I can ping Jorge if needed. 👍